orvara/tests/config_Gl758.ini

```diff
@@ -39,7 +39,7 @@ use_epoch_astrometry = True
 [priors_settings]
 # Priors on primary mass (solar), if prior is not specified, mpri should be inf.
 mpri = 1
-mpri_sig = inf
+mpri_sig = 0.05
```

Comment:
Hi @Yiting206265, this PR looks great. My only question was about this change. Was this a remnant/accidental change? I am not sure where this config file is used (its not in the tests, as far as I can tell). Thanks

Reply:
Just putting this here for myself: Yiting confirmed that this change should be removed before merge.